aiida/orm/nodes/data/structure.py

```diff
@@ -1861,6 +1844,8 @@ def _get_object_pymatgen_structure(self, **kwargs):
 
         if self.pbc != (True, True, True):
             raise ValueError('Periodic boundary conditions must apply in all three dimensions of real space')
+        if self.cell == _DEFAULT_CELL:
+            raise ValueError('Must specify cell in order to create pymatgen structure')
```

Comment:
Note (I put a comment here because I cannot comment above, line 1795: in `asecell = ase.Atoms(cell=self.cell, pbc=self.pbc)`, this works just because `self.cell` is now _DEFAULT_CELL, that is set to `None`; if this changes in the code, that line will break. I suggest to split that line with an if/else, if `self.cell == _DEFAULT_CELL` then  one just calls `ase.Atoms(pbc=self.pbc)` without passing `cell`